### DISCUSSION TITLE
Change speed of animation

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#120] [Help / Questions] [answered]
How would I change the speed of an animation that I'm playing?

**Comment by yandeu** [comment, marked as answer]
https://discourse.threejs.org/t/animation-toggle/12648/4

**Reply by N8python** [reply]
Thank you! Where would I find the `AnimationAction` within the library and modify (after say, a call to `play`).

**Reply by N8python** [reply]
Never mind - I found it. In the `._mixer` attribute of a bodies `animation`.

**Reply by yandeu** [reply]
Better is `object._animationActions.get(name)`.

I just added better support. In the next version you can do:
`object.animation.get(name)`

See [extendedObject3D.ts](https://github.com/enable3d/enable3d/blob/0771e05c328c7c36ec4357f8e035559aa6f004bf/packages/common/src/extendedObject3D.ts#L70).

**Reply by N8python** [reply]
Awesome - thanks for updating the API!
